charts/nginx-ingress-controller/test/default.yaml.out

```diff
@@ -563,39 +568,10 @@ webhooks:
       - v1
     clientConfig:
       service:
-        namespace: "default"
         name: nginx-ingress-controller-admission
+        namespace: default
         path: /networking/v1/ingresses
 ---
-# Source: nginx-ingress-controller/charts/nginx/templates/admission-webhooks/job-patch/networkpolicy.yaml
-apiVersion: networking.k8s.io/v1
-kind: NetworkPolicy
-metadata:
-  name: nginx-ingress-admission
```

Comment:
Yes, the NP was made optional and disabled by default. Since we do not explicitly use/need it, I chose to not deviate from the defaults, but if you think it's bad, we can also (p)reconfigure our nginx chart to enable the NP.

Reply:
ACK, let's omit it given we didn't explicitly need it.